ChangeLog

```diff
@@ -1,3 +1,79 @@
+* Tue Apr 28 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4
+- Changesets: 442
+- Diff: 269 files changed, 21845 insertions(+), 13363 deletions(-)
+
+- Features added since Pacemaker-2.0.3
+  + build: Add support for glib-based unit tests.
+  + build: bumplibs: add git coloring, show PRs merged & show commits
+                     touching each library
+  + controller: add new feature 'shutdown-locks'
+        Optionally allow a gracefully shutdown node to have the resources
+        locked to it for a configurable time.
+        So that it can be restarted with exactly the same resources
+        running as before.
+  + controller/fencer/scheduler: add new feature 'priority-fencing-delay'
+        Optionally derive the priority of a node from the resource-priorities
+        of the resources it is running.
+        In a fencing-race the node with the highest priority has a certain
+        advantage over the others as fencing requests for that node are
+        executed with an additional delay.
+        controlled via cluster option priority-fencing-delay (default = 0)
+  + libcrmcommon: Add a formatter for logs
+  + libpe_status: Add several new functions for working with tags.
+  + stonith_admin: add --delay option (default = 0) to support enforced
+                   fencing delay
+  + tools: Add --include= and --exclude= to crm_mon.
+  + tools: Add --node= to filter crm_mon output for a node (or tag).
+  + tools: Add an option for printing failed resource actions.
+  + tools: Add more fine grained fencing arguments.
+  + tools: Allow crm_rule to check some date_specs.
+
+- Changes since Pacemaker-2.0.3
+  + build: improve checking headers for C++ fitness
+  + build: restore build on latest OpenSUSE
+  + gnublib: refresh to e3b56bc2e68f747320d67754df575976266a5712
+  + libstonithd: Report an error getting history in output instead of
+                 empty history
+  + libstonithd: Improve parameter handling for RHCS-style fence-agents
+                 - make parameter `plug` & `action` shown as not required
+		 - add `plug` parameter besides deprecated `port`
+  + controller: clear leaving node's transient attributes even if there is no DC
+  + controller: don't ack same request twice
+  + cts: Add tests for new crm_rule capabilities.
+  + cts: Improve & Correct tests for date_expressions.
+  + executor: correctly convert nanoseconds to milliseconds
+  + executor: handle systemd execution times under 1 second
```

Comment:
We can drop the above two, they only affected 2.0.3 builds where someone specifically edited configure.ac to comment out a CPPFLAGS line (which was done for fedora rawhide at the time, and was unlikely to be used by users building from source)